poster/src/index.ts

```diff
@@ -14,9 +27,14 @@ async function run() {
     .option('timeout', {alias: 't', description: 'how many seconds to wait before retrying with more gas', type: 'number', default: 180})
     .option('gas-limit', {alias: 'g', description: 'how much gas to send', type: 'number', default: 4000000})
     .option('gas-price', {alias: 'gp', description: 'gas price', type: 'number'})
-    .option('price-delta', {alias: 'd', description: 'the min required difference between new and previous asset price for price update on blockchain', type: 'number', default: 1})
     .option('asset', {alias: 'a', description: 'A list of supported token names for posting prices', type: 'array', default: ['BTC', 'ETH', 'DAI', 'REP', 'ZRX', 'BAT', 'KNC', 'LINK', 'COMP']})
+    // order of `asset` and `price-deltas` should match
+    .option('price-deltas', {
+      alias: 'd', description: 'the min required difference between new and previous asset price for the update on blockchain', type: 'array',
+      default: [defaultDeltas['BTC'], defaultDeltas['ETH'], defaultDeltas['DAI'], defaultDeltas['REP'], defaultDeltas['ZRX'], defaultDeltas['BAT'], defaultDeltas['KNC'], defaultDeltas['LINK'], defaultDeltas['COMP']]
```

Comment:
is it not possible to pass an object here with named values instead of a list? relying on the ordering of these seems kind of error prone

Reply:
then we could skip rebuilding the object below

Reply:
+1

Reply:
I wish, but object type  is not supported by yarg arguments.

````type: one of the following strings
'array': synonymous for array: true, see array()
'boolean': synonymous for boolean: true, see boolean()
'count': synonymous for count: true, see count()
'number': synonymous for number: true, see number()
'string': synonymous for string: true, see string()```

Reply:
I can do string with json object, just realized that